### PR TITLE
[WIP] Code review

### PR DESCRIPTION
**Pull request by EmanueleMinotto**
I'm doing some code reviews (mainly documentation), let me know if everything is ok or not. :)

To Do:
- [ ] PHPDoc for `src/App.php`
- [ ] [Object Calisthenics](https://github.com/object-calisthenics/phpcs-calisthenics-rules) for the classes directly under `GianArb\Penny`
- [ ] Move FQCN (thanks @samsonasik)
- [ ] Apply a quotes convention (I know, I know... I'm pedant)

### CODE REVIEW

**Comment by EmanueleMinotto** [comment]
@gianarb let me know if there's something wrong in the documentation

**Comment by gianarb** [comment]
It sounds perfect only on question src/App.php docs is work in progress? :)

**Comment by EmanueleMinotto** [comment]
yes, I'll update the to do list above

File: src/Dispatcher.php [hunk 1]
```diff
@@ -6,13 +6,33 @@
...
+     */
     private $router;
 
-    public function __construct($router)
+    /**
+     * Class constructor with required FastRoute dispatcher implementation.
+     *
+     * @param \FastRoute\Dispatcher $router Inner router (based on Nikic FastRouter).
+     */
+    public function __construct(\FastRoute\Dispatcher $router)
```
Comment (samsonasik) [inline src/Dispatcher.php:21]: fqcn can be moved to use statement

**Comment by gianarb** [comment]
@EmanueleMinotto can you submit more PRs? :) step by step.. At the moment we are developing quickly and I prefer this flow :dancer: 

Thanks

**Comment by EmanueleMinotto** [comment]
:+1:

**Pull request closed without merging**